apps/www/_blog/2024-11-30-hack-the-base.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Hack the Base! with Supabase'
+description: 'Play cool games, win cool prizes.'
+author: stephen_morgan
+launchweek: '13'
+categories:
+  - launch-week
+tags:
+  - launch-week
+  - security
+date: '2024-12-02'
+---
+
+## Are you ready to test your hacking skills?
```

Comment:
I'd remove this - we shouldn't start the blog post with a title, just directly go into the intro

(or you can add an intro above this)